sgqlc/operation/__init__.py

```diff
@@ -1446,18 +1451,24 @@ def __init__(self, typ=None, name=None, **args):
             name = typ.__name__
 
         self.__type = typ
+        self.__kind = self.__resolve_kind(type, kind)
```

Comment:
is this correct? I believe there is a typo, `typ` is the parameter, while `type` should be the global function.

it happens to work because you don't use this parameter, rather use `self.__type.__name__` inside the function `__resolve_kind()`